src/build.py

```diff
@@ -576,26 +576,42 @@ def GetRepoInfo():
   return info
 
 
+def Which(name):
+  """Find an executable on the system by name. If not found return ''."""
+  # If we want to run this on Windows, we'll have to be smarter.
```

Comment:
Oh, sorry, I forgot about `which` vs. `where`, so I thought you were talking about `check_output`.  Reportedly, certain minor versions of Python 2.7 don't support `check_output` on Windows.